vuepress/src/overview/ix-video.md

```diff
@@ -0,0 +1,293 @@
+# 📹 ix-video
+
+**ix-video** is a web component that makes it quick and easy to use our powerful video processing and delivery platform inside your web application.
+
+The component is built on top of the [video.js](https://videojs.com) library, which is a free, open source, cross-platform video player library for the web that is used by millions of developers.
+
+Features:
+
+[🏎️ Fast, High-Quality Streaming By Default](#basic-usage)
+
+[🎨 Fully customizable out of the box](#customization)
+
+[📝 Support for all HTML5 video formats](#type)
+
+[⚙️ Powerful configuration options](#video-js-configuration)
+
+## Setup
+
+::: tip Note
+
+**HLS** is the default playback format for [imgix video](https://blog.imgix.com/2021/12/15/streaming-video-part-3-http-live-streaming) and the `ix-video` player.
+
+HLS video playback leverages adaptive bitrate streaming **(ABS)** technology to deliver the smooth video quality in the quickest time possible.
+:::
+
+### Frontend Frameworks
+
+If you're using the library in React or Vue, you might have to take some additional steps to get the component to work correctly.
+
+#### Vue
+
+Vue has great support for Web Components, so you can use the component in your Vue application without any additional steps.
+
+However, Vue might show some warnings in the console if you don't declare your custom element in it's configuration. To turn these warnings off, you can add the following:
```

Comment:
```suggestion
You will need to signal to the Vue compiler that this component is a custom element by adding the following:

```js
//vue.config.js
module.exports = {
  chainWebpack: config => {
    config.module
      .rule('vue')
      .use('vue-loader')
      .tap(options => ({
        ...options,
        compilerOptions: {
          // treat any tag that starts with ix- as custom elements
          isCustomElement: (name) => name.startsWith('ix-'),
        }
      })
    )
  }
  
  Or if you're using Vite, you can add the following to your `vite.config.js`:
```

Reply:
On that note, do we still need to include these examples in these files as well? We already include them on the usage page.

Reply:
True, I think they're safe to remove.

Reply:
The usage page wasn't being linked/rendered anywhere 🤦🏼 . Moved its content into the main readme for now.